weaviate/collection/grpc.py

```diff
@@ -754,6 +848,147 @@ def near_text_options(
             )
         ]
 
+    def near_image_flat(
+        self,
+        image: str,
+        certainty: Optional[float] = None,
+        distance: Optional[float] = None,
+        autocut: Optional[int] = None,
+        filters: Optional[_Filters] = None,
+        return_metadata: Optional[MetadataQuery] = None,
+        return_properties: Optional[PROPERTIES] = None,
+    ) -> List[_Object[Properties]]:
+
+        return [
+            self._result_to_object(obj)
+            for obj in self.__create_query().near_image(
+                image=image,
+                certainty=certainty,
+                distance=distance,
+                filters=filters,
+                autocut=autocut,
+                return_metadata=return_metadata,
+                return_properties=return_properties,
+            )
+        ]
+
+    def near_image_options(
+        self,
+        image: str,
+        returns: ReturnValues,
+        options: Optional[NearImageOptions] = None,
+        filters: Optional[_Filters] = None,
+    ) -> List[_Object[Properties]]:
```

Comment:
Testing this, I noticed how you have to specify `returns=ReturnValues()` even if you want everything back. Having this be optional everywhere might be beneficial for UX unless we want users making that a conscious decision